File: internal/content/testdata/bench/rust/split-odd-even-indices.rs
fn split_odd_even(s: String) -> Vec<String> {
    let chars: Vec<char> = s.chars().collect();
    let even: String = chars.iter().step_by(2).collect();
    let odd: String = chars.iter().skip(1).step_by(2).collect();
    vec![even, odd]
}

File: internal/content/testdata/bench/csharp/split-odd-even-indices.cs
public class Solution {
    public string[] split_odd_even(string s) {
        var even = new System.Text.StringBuilder();
        var odd = new System.Text.StringBuilder();
        for (int i = 0; i < s.Length; i++) {
            if (i % 2 == 0) even.Append(s[i]);
            else odd.Append(s[i]);
        }
        return new string[] { even.ToString(), odd.ToString() };
    }
}
